### ISSUE TITLE
trying to get started : domain bug?

### ISSUE THREAD

**Issue by ireneom3** [fluent-ffmpeg/node-fluent-ffmpeg#489]
command line ffmpeg working, but when I try multiple times creating a simple js, using node-express, starting with this : 

var ffmpeg = require('fluent-ffmpeg');
var command =  ffmpeg();

then entering the next few lines from the examples and the main doc after .. I always get this error:

events.js:24
  this.domain = null;
              ^

TypeError: Cannot set property 'domain' of undefined
    at EventEmitter.init (events.js:24:15)
    at EventEmitter (events.js:6:21)
   at FfmpegCommand (fluent file path)
    at Object.<anonymous> (js file path)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
Finished in 0.6s with exit code 1

it also skipped the stdout and stderr outputs.
 
I found a similar project with the same error, and they solved it by just setting 
EventEmitter.(this)  to  EventEmitter.call(this)

no success .. :( also tried every example and reread, logged, restart ... 

any help would be great! thanks!.

specs:

ffmpeg - 2.8.4
fluent-ffmpeg - latest (20/12/15)
OS - win7

**Comment by njoyard** [comment]
Which version of nodejs are you using?
I don't think the problem comes from fluent-ffmpeg there. We're just calling EventEmitter.call(this), which should always work.

**Comment by ireneom3** [comment]
4.2.3 
from https://nodejs.org/en/download/

**Comment by njoyard** [comment]
Can you try this code with your setup?

```js
'use strict';
const util = require('util');
const EventEmitter = require('events');

function MyEventEmitter() {
  // Initialize necessary properties from `EventEmitter` in this instance
  EventEmitter.call(this);
  console.log('OK');
}

// Inherit functions from `EventEmitter`'s prototype
util.inherits(MyEventEmitter, EventEmitter);

var test = new MyEventEmitter();
```

**Comment by ireneom3** [comment]
console prints 'OK'

**Comment by njoyard** [comment]
What about this one ?

```js
'use strict';
const util = require('util');
const EventEmitter = require('events');

function MyEventEmitter() {
  if (!(this instanceof MyEventEmitter)) {
    return new MyEventEmitter();
  }

  // Initialize necessary properties from `EventEmitter` in this instance
  EventEmitter.call(this);
  console.log('OK');
}

// Inherit functions from `EventEmitter`'s prototype
util.inherits(MyEventEmitter, EventEmitter);

var test = MyEventEmitter();
```

**Comment by ireneom3** [comment]
'OK' too

**Comment by njoyard** [comment]
That's weird, because that's exactly what's done in fluent-ffmpeg.
Can you please post your complete code along with the full error stack trace please?

**Comment by ireneom3** [comment]
changes only made in the app.js file, all other files are from the node express template except for the fluent module and json dependancy update. 

https://drive.google.com/file/d/0B93WSGnZ62pJZE1nT3RLTTBCNnc/view?usp=sharing


events.js:24
  this.domain = null;
              ^

TypeError: Cannot set property 'domain' of undefined
    at EventEmitter.init (events.js:24:15)
    at EventEmitter (events.js:6:21)
    at FfmpegCommand (C:\Users\ireneom3\Desktop\nodes-express\node_modules\fluent-ffmpeg\lib\fluent-ffmpeg.js:15:16)
    at Object.<anonymous> (C:\Users\ireneom3\Desktop\nodes-express\app.js:15:15)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
[Finished in 0.8s]

**Comment by njoyard** [comment]
You're using an outdated version of fluent-ffmpeg (1.7.2), please update to the latest version.

**Comment by ireneom3** [comment]
OMG SO HApPy right now .. finally got something working thanks a lot <3<3!
after messing around with everything I seem to have changed and forgot to update the important ones!  finally got passed that error with your help, by simply deleting it from the modules folder!
then I tried going further, because of excitement from going a bit further,  when I tried the merging example I encountered a cannot find ffprobe, when I checked my paths it wasn't there!! I was sure I set those up before...
lack of sleep, really doesn't help! x.x
anyway! thanks a lot again!!
